src/web_app/html/index_template.html

```diff
@@ -28,6 +28,11 @@
   <link rel="canonical" href="{{ room_link }}">
 {% endif %}
   <link rel="stylesheet" href="/css/main.css">
+  <!-- callstats dependencies: http://www.callstats.io/api/#integrating-with-your-sdk -->
+  <script src="https://api.callstats.io/static/callstats.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
+  <script src="https://cdn.socket.io/socket.io-1.2.0.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jsSHA/1.5.0/sha.js"></script>
```

Comment:
callstats.js version 3.10.0 is available now, it deprecates the jquery usage, so the $ as first argument can be removed.

var callStats = new callstats(null,io,jsSHA);

We also added two new fabric events, screenSharingStart and screenSharingStop to track the screen sharing start/stop events during the call.